### PR TITLE
fix: add default env values in docker-compose

### PR DESCRIPTION
**Pull request by AlexanderLukin**
As stated in `README.md`, some environmental variable values are optional and so can be omitted in the `.env` config file. But these variables didn't have default values in the `docker-compose.yml` config file, so, it was not possible to omit these values when the app was built using Docker. Now default values are added for all optional env variables in the `docker-compose.yml`.

### CODE REVIEW

**Comment by AlexanderLukin** [comment]
Add all environmental variables to the `docker-compose.yml`. Improme Readme.

1. Add all environmental variables that are used by the app to the `docker-compose.yml`. Add default values for these variables.

2. Add information about acceptable environmental variables values to the Readme. Add the new "Values" section for each env variable that has a pre-defined set of possible values.

3. Move additional information about environmental variables to the "Note" section in Readme.

4. Add information about the `NODE_ENV` environmental variable to the Readme.

5. Add information about default environmental variable values to the Readme where it was missing.

6. Fix minor issues in Readme. Rephrase some sentences.

7. Add the appropriate Chain ID for the Holesky testnet in the `Network` enum.

File: docker-compose.yml [hunk 1]
```diff
@@ -156,18 +156,41 @@ services:
...
       - CL_API_URLS=${CL_API_URLS}
+      - CL_API_RETRY_DELAY_MS=${CL_API_RETRY_DELAY_MS:-500}
+      - CL_API_GET_RESPONSE_TIMEOUT=${CL_API_GET_RESPONSE_TIMEOUT:-15000}
+      - CL_API_MAX_RETRIES=${CL_API_MAX_RETRIES:-1}
+      - CL_API_GET_BLOCK_INFO_MAX_RETRIES=${CL_API_GET_BLOCK_INFO_MAX_RETRIES:-1}
+      - FETCH_INTERVAL_SLOTS=${FETCH_INTERVAL_SLOTS:-32}
+      - CHAIN_SLOT_TIME_SECONDS=${CHAIN_SLOT_TIME_SECONDS:-12}
+      - WORKING_MODE=${WORKING_MODE:-finalized}
+      - NODE_ENV=${NODE_ENV:-development}
+      - DRY_RUN=${DRY_RUN:-false}
```
Comment (vgorkavenko) [inline docker-compose.yml:187]: let's move them up. very important vars

File: src/common/config/env.validation.ts [hunk 2]
```diff
@@ -1,4 +1,7 @@
-import { Transform, plainToInstance } from 'class-transformer';
+import {
```
Comment (vgorkavenko) [inline src/common/config/env.validation.ts:1]: please run `yarn format`

File: docker-compose.yml (hunk 1 above)
Comment (AlexanderLukin) [reply docker-compose.yml]: Agree. Moved these vars to the beginning of the `environment` section.

**Review by vgorkavenko — APPROVED** [review]

File: src/common/config/env.validation.ts (hunk 2 above)
Comment (AlexanderLukin) [reply src/common/config/env.validation.ts]: Sure. Will use `prettier` to reformat code before each new commit next time.

**Pull request merged**